frontends/main/src/app-pages/Articles/NewArticlePage.tsx

```diff
@@ -0,0 +1,122 @@
+"use client"
+import React, { useState, ChangeEvent } from "react"
+import { Permission } from "api/hooks/user"
+import { useRouter } from "next-nprogress-bar"
+import { useArticleCreate } from "api/hooks/articles"
+import { Button, Input, Alert } from "@mitodl/smoot-design"
+import RestrictedRoute from "@/components/RestrictedRoute/RestrictedRoute"
+import { Container, Typography, styled } from "ol-components"
+
+const SaveButton = styled.div({
+  textAlign: "right",
+  margin: "10px",
+})
+
+const ClientContainer = styled.div({
+  width: "100%",
+  margin: "10px 0",
+})
+
+const TitleInput = styled(Input)({
+  width: "100%",
+  margin: "10px 0",
+})
+
+const NewArticlePage: React.FC = () => {
+  const router = useRouter()
+
+  const [title, setTitle] = React.useState<string>("")
+  const [text, setText] = useState("")
+  const [json, setJson] = useState({})
+  const [alertText, setAlertText] = React.useState("")
+  const [severity, setSeverity] = React.useState<"success" | "error">("success")
+
+  const { mutate: createArticle, isPending } = useArticleCreate()
+
+  const handleSave = () => {
+    setAlertText("")
+
+    const payload = {
+      title: title.trim(),
+      content: json,
+    }
+
+    createArticle(
+      payload as {
+        content: object
+        title: string
+      },
+      {
+        onSuccess: (article) => {
+          router.push(`/articles/${article.id}`)
```

Comment:
Similar to https://github.com/mitodl/mit-learn/pull/2686/files#r2505090921, use `articlesView(id)`.